knip.jsonc

```
@@ -2,5 +2,5 @@
 	"$schema": "https://unpkg.com/knip@latest/schema.json",
 	"entry": ["src/index.ts!", "src/cli.ts!"],
 	"ignoreExportsUsedInFile": { "interface": true, "type": true },
-	"project": ["src/**/*.ts!"]
+	"project": ["src/**/*.ts!"],
```

Unrelated change - I'm guessing some tooling issue automatically did it? That would be something broken to fix outside of this PR.

```suggestion
	"project": ["src/**/*.ts!"]
```